.ci/tav.json

```diff
@@ -1,39 +1,38 @@
 {
-  "// todo": "We want versions=['20','19','18','16','14','12','10','8'], but versions*modules needs to be <256 for the GH Actions jobs limit",
-  "versions": [ "20", "18", "16", "14", "12", "10", "8" ],
+  "versions": [ 20, 19, 18, 16, 14, 12, 10, 8 ],
   "modules": [
-    "@apollo/server",
-    "@aws-sdk/client-s3",
-    "@elastic/elasticsearch",
-    "@hapi/hapi",
-    "@opentelemetry/api",
-    "@opentelemetry/sdk-metrics",
-    "apollo-server-express",
-    "aws-sdk",
-    "cassandra-driver",
-    "elasticsearch",
-    "express",
-    "express-queue",
-    "fastify",
-    "finalhandler",
-    "generic-pool",
-    "graphql",
-    "ioredis",
-    "knex",
-    "memcached",
-    "mongodb",
-    "mongodb-core",
-    "mysql",
-    "mysql2",
-    "next",
-    "pg",
-    "redis",
-    "restify",
-    "tedious",
-    "undici",
-    "ws",
-    "@koa/router,koa-router",
-    "handlebars,pug",
-    "bluebird,got"
+    {"name": "@apollo/server", "minVersion": 14 },
+    {"name": "@aws-sdk/client-s3", "minVersion": 14 },
+    {"name": "@elastic/elasticsearch", "minVersion": 10 },
+    {"name": "@hapi/hapi", "minVersion": 8 },
+    {"name": "@opentelemetry/api", "minVersion": 14 },
+    {"name": "@opentelemetry/sdk-metrics", "minVersion": 14 },
+    {"name": "apollo-server-express", "minVersion": 8 },
+    {"name": "aws-sdk", "minVersion": 8 },
+    {"name": "cassandra-driver", "minVersion": 8 },
+    {"name": "elasticsearch", "minVersion": 8 },
+    {"name": "express", "minVersion": 8 },
+    {"name": "express-queue", "minVersion": 8 },
+    {"name": "fastify", "minVersion": 8 },
+    {"name": "finalhandler", "minVersion": 8 },
+    {"name": "generic-pool", "minVersion": 8 },
+    {"name": "graphql", "minVersion": 8 },
+    {"name": "ioredis", "minVersion": 8 },
+    {"name": "knex", "minVersion": 8 },
+    {"name": "memcached", "minVersion": 8 },
+    {"name": "mongodb", "minVersion": 8 },
+    {"name": "mongodb-core", "minVersion": 8 },
+    {"name": "mysql", "minVersion": 8 },
+    {"name": "mysql2", "minVersion": 8 },
+    {"name": "next", "minVersion": 14 },
+    {"name": "pg", "minVersion": 8 },
+    {"name": "redis", "minVersion": 8 },
+    {"name": "restify", "minVersion": 14 },
+    {"name": "tedious", "minVersion": 8 },
+    {"name": "undici", "minVersion": 8 },
+    {"name": "ws", "minVersion": 8 },
+    {"name": "@koa/router,koa-router", "minVersion": 8 },
+    {"name": "handlebars,pug", "minVersion": 8 },
+    {"name": "bluebird,got", "minVersion": 8 }
```

Comment:
Hrm, I guess with all these `minVersion: 8`s, it means there won't be a large reduction in the number of jobs. Still probably worth keeping.

(Not part of this PR, but I wonder if we could consider dropping testing for versions of these packages that are, say, older than 2y and not of the latest major version. That might enable raising the `minVersion` here.)

Reply:
Maybe we can leverage the multiple package option we have, here we are testing 2 packages in the same job
```
{"name": "bluebird,got", "minVersion": 8 }
```
This is actually what I want to do for `@aws-sdk/client-*` to test all of them at once for each version. There might be other groups we could make perhaps `@opentelemetry/*` or `mysql`? Of course some test review is needed to see compatibility. Also I think the `runTestFixtures` util introduced in the [ESM support PR](https://github.com/elastic/apm-agent-nodejs/pull/3381/files#diff-4044438594f145ca4e5b9920e844237cce63dd56744a23e3f263456e6a9f7d35R246) would come handy